src/ZPublisher/HTTPRequest.py

```diff
@@ -1644,18 +1644,19 @@ class TemporaryFileWrapper(_TemporaryFileWrapper):
     and doesn't complain about such a move either.
     """
 
-    unlink = staticmethod(unlink)
-    isfile = staticmethod(isfile)
+    if PY2:
+        unlink = staticmethod(unlink)
+        isfile = staticmethod(isfile)
```

Comment:
Why are these made `staticmethods`?  They could just be called directly (even as `os.unlink` and `os.path.isfile`).